### ISSUE TITLE
[Suggestion]: Make version without fancy clouds for M1 Mac or make a serverside version

### ISSUE THREAD

**Issue by souldbminerr** [nonamecrackers2/simple-clouds#80]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Hello!
I use macOS sometimes, and I would love to use this mod on my macs.
Is it possible to ether make a version without fancy clouds, or a server side version?

**Comment by 9182736450typist** [comment]
I agree. However, you can always use ASAHI LINUX on an M series mac or Parallels/Bootcamp on an intel mac to run OpenGL 4.2-4.6 and the mod will probably work.

**Comment by rinOfTheStars** [comment]
> I agree. However, you can always use ASAHI LINUX on an M series mac or Parallels/Bootcamp on an intel mac to run OpenGL 4.2-4.6 and the mod will probably work.

I can confirm that it *works* on Asahi Linux, but the performance cost is surprisingly steep—this might just be due to my machine (M1 Mac Mini) being a few years old at this point, though.

**Comment by nonamecrackers2** [comment]
Not really, the entire core of how this mod works is compute shaders which is what is not available on most macs. A not-so-fancy version wouldn't be very feasible to implement whatever technique I decide to go with and I don't really have the time to work on it. I'm incredibly sorry it doesn't work on Macs and will look into work arounds to fix this issue, but otherwise there isn't much I can do.

Also what are meaning with a server-side version? The jar can already be used on the server-side

**Comment by souldbminerr** [comment]
I mean the weather works clientside with packet manipulation, as the mod if installed on server requires it to be installed on client

**Comment by nonamecrackers2** [comment]
Oh I see what you mean. I could probably consider doing that.
